app/models/enhancements/application.rb

```diff
@@ -1,12 +1,26 @@
 require "doorkeeper/orm/active_record/application"
 
 class ::Doorkeeper::Application < ActiveRecord::Base
+  include ActiveModel::ForbiddenAttributesProtection
+
   has_many :permissions, :dependent => :destroy
   has_many :supported_permissions, :dependent => :destroy
 
-
-  attr_accessible :name, :description, :uid, :secret, :redirect_uri, :home_uri
-  attr_accessible :supports_push_updates, role: :superadmin
+  # TODO: Remove this accessible attributes list once Signonotron has upgraded
```

Comment:
We worked around a similar problem in the strong parameters upgrade on Whitehall by making `attr_accessible` a no-op: https://github.com/alphagov/whitehall/commit/a13a719118048c3fb296706a857846847b47ad07

I think using a similar approach here would let us drop the `attr_accessible` call.